README.md

```diff
@@ -1,253 +1,147 @@
-# Mini Deal Bot
+# Deal Bot
 
-<p>
-  An automated deal-making bot for the Filecoin Calibration Network built with NestJS and Synapse SDK
-</p>
+> Automated Filecoin storage deal creation and performance monitoring system
 
-## Description
+[![Node](https://img.shields.io/badge/node-%3E%3D18-brightgreen.svg)](https://nodejs.org)
+[![TypeScript](https://img.shields.io/badge/typescript-5.x-blue.svg)](https://www.typescriptlang.org)
 
-FWSS Deal Bot is an intelligent automation system that creates storage deals on the Filecoin Network every 30 minutes. It features automated deal creation across multiple storage providers, CDN A/B testing, comprehensive metrics tracking, and uses local datasets for testing storage and retrieval performance.
+An intelligent automation system for creating and monitoring storage deals on the Filecoin network. Features automated deal creation, CDN performance testing, comprehensive metrics tracking, and a modern web dashboard.
```

Comment:
Maybe change to ”monitoring PDP deals on the Filecoin network" so that it's a little bit more specific